editoast/src/chartos/layer_cache.rs

```diff
@@ -92,13 +84,31 @@ pub fn count_tiles(max_zoom: u64, bbox: &BoundingBox) -> u64 {
     count
 }
 
+pub fn get_layer_cache_prefix(layer_name: &str, infra_id: i32) -> String {
+    format!("chartis.layer.{layer_name}.infra_{infra_id}")
```

Comment:
```suggestion
    format!("editoast.layer.{layer_name}.infra_{infra_id}")
```

Or `chartos` but I think we should stop using this name.

Reply:
I will do that once chartos will not be responsibe to cache tiles in the redis (I have to do #2893)